src/hyperloglog.c

```diff
@@ -1363,12 +1607,16 @@ void pfmergeCommand(client *c) {
 
     /* Write the resulting HLL to the destination HLL registers and
      * invalidate the cached value. */
-    for (j = 0; j < HLL_REGISTERS; j++) {
-        if (max[j] == 0) continue;
-        hdr = o->ptr;
-        switch (hdr->encoding) {
-        case HLL_DENSE: hllDenseSet(hdr->registers, j, max[j]); break;
-        case HLL_SPARSE: hllSparseSet(o, j, max[j]); break;
+    if (use_dense) {
+        hllDenseCompress(hdr->registers, max);
+    } else {
+        for (j = 0; j < HLL_REGISTERS; j++) {
+            if (max[j] == 0) continue;
+            hdr = o->ptr;
+            switch (hdr->encoding) {
+            case HLL_DENSE: hllDenseSet(hdr->registers, j, max[j]); break;
+            case HLL_SPARSE: hllSparseSet(o, j, max[j]); break;
```

Comment:
Nit: Missing indentation.

Reply:
clang-format-18 removes the indentation 😂

Reply:
Yes, `case` and labels for goto are indented one step less than other code. This is correct indentation, a common style in C.